pkg/api/v1beta1/awspcaissuer_types.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package v1beta1
 
 import (
-	v1 "k8s.io/api/core/v1"
+	core "k8s.io/api/core/v1"
```

Comment:
Any reason why this changed from `v1`?

Reply:
This was unintentional, I'll remove so there's not an extra diff. Some of the tests do import this as v1, so I must have mixed it up.